### PR TITLE
hgtool: Adding initial implementation

### PR DESCRIPTION
**Pull request by simar7**
Still a WIP. Don't merge.

Signed-off-by: Simarpreet Singh <s244sing@uwaterloo.ca>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from distutils.core import setup
+from setuptools import find_packages
+
+setup(
+    name='hgtool',
+    version='1.0.0',
+    author='Chris AtLee',
+    author_email='catlee@mozilla.com',
+    packages=find_packages(),
+    url='https://hg.mozilla.org/build/tools/file/default/hgtool',
```
Comment (simar7) [inline setup.py]: What should the updated URL be?

**Comment by simar7** [comment]
So with the rework needed in order to move _rmtree_windows() into mozfile, this would be the overhead in terms of modifying mozfile.

**Comment by simar7** [comment]
On a side note, can someone with admin access create this PR into an issue and assign it to me? Thanks!

File: setup.py (hunk 1 above)
Comment (armenzg) [inline setup.py]: For now, let's point here, however, we never know where we will end up :)

**Comment by armenzg** [comment]
I tried assigning you to the issue but "simar7" is not showing up:
https://github.com/mozilla/hgtool/issues/2

**Comment by armenzg** [comment]
Let's move into mozfile only what applies and not move into it the functions from commands.py.

Does that make sense?

**Comment by simar7** [comment]
Based on IRC discussions, plan for now is to leave mozfile as it is and leverage functionality from commands.py until there comes a point when it is a good idea to move things into mozfile.

**Comment by simar7** [comment]
Closing this as decisions changed again to move it back to version-control-tools.

**Pull request closed without merging**